RecoLocalTracker/SubCollectionProducers/src/JetCoreClusterSplitter.cc

```diff
@@ -138,20 +137,20 @@ void JetCoreClusterSplitter::produce(edm::Event& iEvent,
             // check if the cluster has to be splitted
 
             bool isEndCap =
-                (fabs(cPos.z()) > 30);  // FIXME: check detID instead!
+                (std::abs(cPos.z()) > 30.f);  // FIXME: check detID instead!
             float jetZOverRho = jet.momentum().Z() / jet.momentum().Rho();
             if (isEndCap)
               jetZOverRho = jet.momentum().Rho() / jet.momentum().Z();
             float expSizeY =
-                fabs(sqrt(1.3 * 1.3 + 1.9 * 1.9 * jetZOverRho * jetZOverRho));
-            if (expSizeY < 1) expSizeY = 1.;
-            float expSizeX = 1.5;
+                std::sqrt((1.3f*1.3f) + (1.9f*1.9f) * jetZOverRho*jetZOverRho);
```

Comment:
do not understand the comment: as I sais this class needs anyhow a major cleanup